### PR TITLE
[WIP] Animal sounds

### PR DESCRIPTION
**Pull request by AlvCyktor**
# Document the changes in your pull request
Adds sounds that will play when some animals speak or die

# Spriting
No Sprites

# Wiki Documentation
No need most likely

# Changelog
:cl:  
rscadd: Added new sounds for some passive animals
rscadd: death and speaking audio variable for simple animals
/:cl:

### CODE REVIEW

**Comment by Moltijoe** [comment]
Hold up, speak sounds?

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -89,6 +89,11 @@
 
 	var/shouldwakeup = FALSE //convenience var for forcibly waking up an idling AI on next check.
 
+	//Audio
+	var/speak_audio = "" //the sound that will play when the animal speaks
```
Comment (ToasterBiome) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: these should be null instead of empty strings

**Comment by AlvCyktor** [comment]
> Hold up, speak sounds?

like when the animal sometimes says stuff like "MOOOOOOOOO" or "BAAAAAAAAAAA"

File: code/modules/mob/living/simple_animal/simple_animal.dm (hunk 1 above)
Comment (MajManatee) [reply code/modules/mob/living/simple_animal/simple_animal.dm]: put an audio clip of you saying "uh oh" as the default sound

**Comment by AlvCyktor** [comment]
https://user-images.githubusercontent.com/81328245/222254473-15dc27bd-bf63-47c9-8754-2cfaaf5960a2.mp4

**Comment by MajManatee** [comment]
minecraft cow noise

**Comment by AlvCyktor** [comment]
Ill continue this when I feel like going on a internet safari to find good sound effects for free

**Pull request closed without merging**